libpod/runtime.go

```diff
@@ -99,6 +99,11 @@ func NewRuntime(options ...RuntimeOption) (*Runtime, error) {
 	return runtime, nil
 }
 
+// Store returns the storage store associated with the runtime
+func (r *Runtime) Store() storage.Store {
```

Comment:
I really, really do not want to do this, even temporarily. Where do we need this?

Reply:
No longer necessary.